site/guide/pastebin.md

```diff
@@ -145,8 +145,9 @@ pub struct PasteID<'a>(Cow<'a, str>);
 impl<'a> PasteID<'a> {
     /// Generate a _probably_ unique ID with `size` characters. For readability,
     /// the characters used are from the sets [0-9], [A-Z], [a-z]. The
-    /// probability of a collision depends on the value of `size`. In
-    /// particular, the probability of a collision is 1/62^(size).
+    /// probability of a collision depends on the value of `size` and
+    /// the amount of IDs generated. If you want to learn more, this is known as
```

Comment:
Change "amount" to "number". Let's say something like:

"The probability of a collision depends on the value of `size` and the number of IDs generated thus far." I think it would be fine if this is all we said about the probability, i.e, end the doc comment there. What do you think?